src/com/plantpocalypse/model/items/NPC.java

```diff
@@ -0,0 +1,30 @@
+package com.plantpocalypse.model.items;
+
+public class NPC {
+    private String name;
+
+
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public NPC(String name) {
+        this.name = name;
+
+    }
+
+    public String respond(){
```

Comment:
When NPCReader class and NPC.xml are updated, please reference the individual NPC's responses here instead of hard coding